### PR TITLE
Enhancement/generalize email templating

### PR DESCRIPTION
**Pull request by Wardormeur**
Requires to : 
 - deploy to master as well
 - change version of email template in sendgrid

### CODE REVIEW

File: server/models/event.js [hunk 1]
```diff
@@ -1,10 +1,14 @@
 const bookshelf = require('../database');
+const moment = require('moment');
 
 const Event = bookshelf.Model.extend({
   tableName: 'event',
   projects() {
     return this.hasMany('Event');
   },
+  formattedDate() {
+    return moment(this.attributes.date).format('dddd [the] Do [of] MMMM');
```
Comment (Wardormeur) [inline server/models/event.js]: Doesn't that implies local time @DanielBrierton ?

Comment (DanielBrierton) [reply server/models/event.js]: Doesn't display time, so yeah, I think this is fine.

Comment (Wardormeur) [reply server/models/event.js]: Add tz

**Comment by Wardormeur** [comment]
Yes, it should

**Pull request merged**